event-store-client/readmodels.md

```diff
@@ -108,9 +117,9 @@ class UserEventAppeared
         ?int $retryCount = null
     ): Promise
     {
-        switch ($resolvedEvent->originalEvent()->eventType()) {
+        switch ($resolvedEvent->event()->eventType()) {
             case 'user-registered':
-                $data = $resolvedEvent->originalEvent()->data();
+                $data = $resolvedEvent->event()->data();
```

Comment:
fyi: the original event is the link-event, because that's what you subscribed to.

Reply:
I finally got it but wow, was hard to understand. (since there's almost no doc about it)